### ISSUE TITLE
Missing Array.find error in IE11

### ISSUE THREAD

**Issue by smathson** [theKashey/focus-lock#9]
I see PR #4 removed usages of Array.find to make this library safe for older browsers without using a polyfill. Looks like a new usage made it's way back in here: https://github.com/theKashey/focus-lock/blob/master/src/focusMerge.js#L102

Is no polyfills for older browsers still a goal of this library? If so, I'm happy to put a PR together to resolve this issue. Thanks!

**Comment by theKashey** [comment]
Please help me with it If you can easily test the result. I recently migrated to a new laptop, and lost all the virtual machines I've had.

Probably the following code would work, and would work better than the existing one:
```js
const reorderNodes = (srcNodes, dstNodes) => {
  const dest = new Set();
  dstNodes.forEach(node => dest.add(node)); // no Map(dstNodes) for IE11 :(
  return  srcNodes.filter(node => dest.has(node);
};
```

**Comment by theKashey** [comment]
released in focus-lock@0.6.5
